libs/subliminal_patch/core.py

```diff
@@ -186,12 +186,9 @@ def list_subtitles_provider(self, provider, video, languages):
         except (requests.Timeout, socket.timeout):
             logger.error('Provider %r timed out', provider)
 
-        except (TooManyRequests, DownloadLimitExceeded, ServiceUnavailable, APIThrottled), e:
-            self.throttle_callback(provider, e)
-            return
-
-        except:
+        except (Exception), e:
```

Comment:
Syntax unsupported in Python 3. Must be `except (Exception) as e:`

Reply:
Fixed